README.md

```diff
@@ -1,3 +1,12 @@
+**Deprecation Notice**: After much discussion among maintainers, we have decided to deprecate this
+repo in favor of the new [wasmCloud Helm Chart](https://github.com/wasmCloud/wasmcloud-otp/tree/main/wasmcloud_host/chart).
+We arrived a this decision due to 2 main reasons. First, the amount of work to bring this up to date
+with both Krustlet 1.0 and the shiny new 0.50+ wasmCloud host is more than we are able to commit
+to at this time. More importantly, running your wasmCloud actors from a Kubernetes context severely
```

Comment:
maybe "running your wasmCloud actors on wasmCloud-specific nodes" rather than "kubernetes context", because we say in the following paragraph that running as containers/pods in k8s is fine.